oxidation/libparsec/crates/serialization_format/src/parser/protocol.rs

```diff
@@ -285,6 +287,17 @@ impl Cmds {
                     #[serde(tag = "status")]
                     pub enum Rep {
                         #variants_rep
+                        #[serde(skip)]
+                        UnknownStatus {
+                            _status: String,
+                            reason: Option<String>,
+                        }
```

Comment:
```suggestion
                        UnknownStatus(UnknownStatus)
```

why not directly use the struct `UnknownStatus` ?

Reply:
Because `status` field is already used in serialization/deserialization by the tag

Reply:
Well but, serde skip this variant (`#[serde(skip)]`), and we already have the value with `let data = ::rmp_serde::from_slice::<UnknownStatus>(buf)?;`